Source/WebCore/page/ScreenOrientationLockType.h

```diff
@@ -37,7 +37,8 @@ enum class ScreenOrientationLockType : uint8_t {
     PortraitPrimary,
     PortraitSecondary,
     LandscapePrimary,
-    LandscapeSecondary
+    LandscapeSecondary,
+    Undefined
```

Comment:
I don’t think we should be adding an enum value. Have your parser return a std::optional if you don’t want it to return ANY in case of parsing failure.

Reply:
Ok, yeah. I saw there was other cases that were using `Undefined` so copied that. std::optional would be better.